crates/rabbitmq/src/queue_type.rs

```diff
@@ -90,7 +90,7 @@ impl<'a> QueueInfo<'a> {
                 Binding::Fanout => ExchangeKind::Fanout,
                 Binding::Direct(_) => ExchangeKind::Direct,
             },
-            ExchangeDeclareOptions::default(),
+            ExchangeDeclareOptions { durable: true, ..ExchangeDeclareOptions::default() },
```

Comment:
I usually see this written as `Default::default()`, which does the same thing, perhaps making it clearer that its using the trait

Reply:
no need to fix tho